src/scoped.component.test.js

```diff
@@ -148,6 +148,75 @@ describe("<Scoped />", function() {
     ReactDOM.unmountComponentAtNode(el);
   });
 
+  it("should create local CSS for combined CSS statements and new lines", function() {
+    const css = `
+      & .wow,
+      & .someRule {
+        background-color: red;
+      }
+    `;
+
+    const el = document.createElement("div");
+
+    ReactDOM.render(
+      <div>
+        <Scoped css={css}>
+          <div>Hello</div>
+        </Scoped>
+      </div>,
+      el
+    );
+
+    expect(document.querySelectorAll(`style[type="text/css"]`)[0].innerHTML.trim()).toBe(
+      `
+      [data-kremling="0"] .wow, [data-kremling="0"].wow, [data-kremling="0"] .someRule, [data-kremling="0"].someRule {
+        background-color: red;
+      }
+    `.trim()
+    );
+
+    ReactDOM.unmountComponentAtNode(el);
+  });
+
+  it("should create local CSS multiple css statements", function() {
```

Comment:
I added this test to make sure the regex isn't doing weird matching across unrelated rules